examples/presigned_post_policy.py

```diff
@@ -25,19 +26,36 @@
     secret_key="zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG",
 )
 
+# In play make sure my-bucket exist if not please create it
 policy = PostPolicy(
     "my-bucket", datetime.utcnow() + timedelta(days=10),
 )
-policy.add_starts_with_condition("key", "my/object/prefix/")
+# key=my/object/prefix/my-object-name should go in the policy and in the filled-in form
+# Same as in our Java example for the example to actually works.
+policy.add_starts_with_condition("key", "my/object/prefix/my-object-name")
 policy.add_content_length_range_condition(1*1024*1024, 10*1024*1024)
 
 form_data = client.presigned_post_policy(policy)
 
+# Decode from bytes so POST will work by having the policy as policy=eyJleHBpcm
+# If policy is like "policy=b'eyJleHBpcm..." Then this will fail.
+if isinstance(form_data['policy'], (bytes, bytearray)):
+    form_data['policy'] = form_data['policy'].decode("utf-8")
+
+# Create the file for the example
+# 2 MB Object to be in the defined range above in the policy.
+with open("output.txt", "wb") as out:
+    out.truncate(1 * 1024 * 1024)
+
+# This example is made with text file, please change file and content type accordingly for an image.
 curl_cmd = (
     "curl -X POST "
     "https://play.min.io/my-bucket "
-    "{0} -F file=@<FILE>"
+    "{0} -F file=@output.txt -F key=my/object/prefix/my-object-name -F Content-Type=text/html -v"
 ).format(
     " ".join(["-F {0}={1}".format(k, v) for k, v in form_data.items()]),
 )
 print(curl_cmd)
+
+# Execute the curl for the user:
+os.system(curl_cmd)
```

Comment:
The idea of this example is how to use `presigned_post_policy()` API. If executing a POST call wih form data is needed, you could use `urllib3` than external `curl` binary.